### PR TITLE
Some step-by-step guides for the new crafting system

### PR DESCRIPTION
**Pull request by Lizurt**
### Purpose

Adds the small step-by-step guides for the new crafting system.

### Notes

Wasn't tested because I don't even know how to open this `.md` file in a browser.

Please check my grammar on the page. English is definitely not my native language.

### CODE REVIEW

File: docs/development/Crafting-System.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+1. Open the Unity Editor 
+2. Follow the path: /Assets/ScriptableObjects/Crafting/Recipes/
+3. Find a necessarry category to put a new recipe into.
+4. RMB -> Create -> ScriptableObjects -> Crafting -> CraftingRecipe
+5. Give the new recipe a name. 
+6. Press the "Add to the singleton if necessary" button.
+7. Edit your new recipe freely(tooltips will help you).
+8. Add the opportunity for a player to learn this recipe.
+  * You can add the recipe to the Player's default known recipes list in the PlayerCrafting component.
```
Comment (NoooneyDude) [inline docs/development/Crafting-System.md]: Might want to specify where the `PlayerCrafting` component is.

Though, discussion in Discord suggests moving this into an SO for creating recipe collections, and then having a list of these recipe collections defined on the occupation SOs, at some point. Something like that.

Comment (Lizurt) [reply docs/development/Crafting-System.md]: > Might want to specify where the PlayerCrafting component is.

Well it's already specified or something like that:

> ... to the Player's ...

Do you mean it's better to write `... to the Player_V4's ...`?

Comment (NoooneyDude) [reply docs/development/Crafting-System.md]: Yeah, that it is on the player prefab.

Comment (Lizurt) [reply docs/development/Crafting-System.md]: > Though, discussion in Discord suggests moving this into an SO for creating recipe collections, and then having a list of these recipe collections defined on the occupation SOs, at some point. Something like that.

I think we may edit the documentation when we implement this.

Comment (NoooneyDude) [reply docs/development/Crafting-System.md]: Yep, outside the scope of this one.

**Pull request merged**